docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1604,9 +1610,15 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/resources/open`|true|
 |`fhirServer/resources/Resource/interactions`|null (all interactions supported)|
 |`fhirServer/resources/Resource/searchParameters`|null (all global search parameters supported)|
+|`fhirServer/resources/Resource/searchIncludes`|null (all \_include values supported)|
+|`fhirServer/resources/Resource/searchRevIncludes`|null (all \_revinclude values supported)|
```

Comment:
What do the `\` do or reference?

Reply:
I assume it just escapes the `_` char.  not sure if its needed or not, but it looks fine at https://github.com/IBM/FHIR/blob/tbieste-issue-1351/docs/src/pages/guides/FHIRServerUsersGuide.md

Reply:
They are used to escape the underscore '_' character in markdown in cases where you want to display the literal.

Reply:
oh snazy - I hadn't even thought about the implication on formatting.  good catch.